### PR TITLE
[mqtt.homeassistant] don't set range on current temp channel

### PR DESCRIPTION
**Pull request by ccutrer**
the min/max from the config are specifically only for the set point. i.e. for a heater in winter when the heater is off, it may have a set point range of 65-104°F, but if it's off for an unoccopied room the ambient temperature might drop far below 65

### CODE REVIEW

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
Let's wait for @antroids approval.

**Comment by antroids** [comment]
Good catch! These values are for command topics only.
LGTM

**Pull request merged**
